File: go/fib.go
package main
import "fmt"

func fib(x int) int {
	if x<2 {
		return 1
	} else {
		return fib(x-1) + fib(x-2)
	}
	return 0
}

func main() {
	fmt.Println(fib(45))
}

File: cs/Fib.cs
using System;

class Fib {
  static void Main() {
    fib( 40 );
  }

  static int fib( int i ) {
    if( i < 2 ) {
      return 1;
    } else {
       return fib( i - 1 ) + fib( i - 2 );
    }
  }
}
